internal/cmd/run/cmd.go

```diff
@@ -43,14 +44,16 @@ func NewCmd() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "run",
-		Short: "Run the operator",
-		Run: func(cmd *cobra.Command, _ []string) {
-			logf.SetLogger(zapl.New(zapl.UseFlagOptions(opts)))
-			r.run(cmd)
-		},
+		Short: "Run the helm operator controller",
 	}
 	r.bindFlags(cmd.Flags())
 	cmd.Flags().AddGoFlagSet(zapfs)
+	cmd.Run = func(cmd *cobra.Command, _ []string) {
+		klog.InitFlags(flag.CommandLine)
```

Comment:
I used this flag initialization, to default to flags set by cli controller, in case 2 controllers for the same operator are launched (one with `make deploy` and the other with cli), but on testing I found that both controllers get stuck on  `Reconcile Errors` `"error":"rendered manifests contain a resource that already exists.`
but one of them(randomly) also carries out reconciler tasks(install/delete) in between printing the error.

So I will revert this change and since this is not a valid behavior for the sdk, we should not need it here.